### ISSUE TITLE
swift compiler error

### ISSUE THREAD

**Issue by hulikp** [marmelroy/FileBrowser#22]
Hello 

I am getting this error use of unresolved identifier 'PreviewManager'.

I have swift 2.3 and filebrowser (0.1.4)

**Comment by hulikp** [comment]
i upgrade to 0.2.0 but got many more errors so i cam back to 0.1.4 and now i have:
Type of 'previewItemURL' has different optionality than required by protocol 'QLpreviewItem'

**Comment by giacDaddy** [comment]
Yeah I got a total of 79 errors ever since the new swift changes came out. I think this app is a dud until it's fixed

**Comment by marmelroy** [comment]
Hi @hulikp and @giacDaddy, sorry to hear about your issues but FileBrowser currently only supports Swift 3.0 (released over a month ago). 

If you are having issues using with Swift 3.0 please let me know.

**Comment by StefanoD** [comment]
Why was there only a Swift 2.2 release, but no Swift 2.3 release?!

**Comment by hulikp** [comment]
Thank you @marmelroy  for your answer that was i thought, @StefanoD there was a 2.3 that was the current version but now it's upgrade to swift 3.

**Comment by hulikp** [comment]
@giacDaddy @StefanoD I found a fork for swift 2.3 works fine from nuglif

**Comment by giacDaddy** [comment]
@hulikp can you link to the fork or briefly demonstrate how you got it to work? Thanks

**Comment by StefanoD** [comment]
@hulikp : Yes, a link to the fork would be cool.

**Comment by hulikp** [comment]
You should addd this to your Podfile @giacDaddy @StefanoD 
pod 'FileBrowser', :git => 'https://github.com/Nuglif/FileBrowser.git', :branch => 'swift2.3'

**Comment by yangkuku** [comment]
@hulikp thank you, I'm looking for this!
